Cargo.toml

```diff
@@ -1694,6 +1694,17 @@ description = "Demonstrates various methods to load assets"
 category = "Assets"
 wasm = false
 
+[[example]]
+name = "arc_asset"
+path = "examples/asset/arc_asset.rs"
+doc-scrape-examples = true
+
+[package.metadata.example.arc_asset]
+name = "Arc'd Assets"
+description = "Demonstrates how to acquire Arc'd assets and use them in an async context"
+category = "Assets"
+wasm = false
+
```

Comment:
the focus of the example should not be "arc_asset", it should be "async_access_to_asset" or something. arc is the technical detail, not the interesting feature

Reply:
Good point! Done.